packages/terra-list/tests/nightwatch/selectable-list/NonSelectableItemSelectableList.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+
+import SelectableList from '../../../src/SelectableList';
+
+class listExample extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { selectedIndex: 'none' };
+    this.handleSelection = this.handleSelection.bind(this);
+  }
+
+  handleSelection(event, selectedIndex) {
+    this.setState({ selectedIndex });
+  }
+
+  render() {
+    return (
```

Comment:
Add context to indicate which item is non-selectable

Reply:
Updated Item name.